NEWS.md

```diff
@@ -8,6 +8,8 @@ shiny 1.5.0.9000
 
 * Added appropriate labels to `icon()` element to provide screen-reader users with alternative descriptions for the `fontawesome` and `glyphicon`: `aria-label` is automatically applied based on the fontawesome name. For example, `icon("calendar")` will be announced as "calendar icon" to screen readers. "presentation" aria role has also been attached to `icon()` to remove redundant semantic info for screen readers.
 
+* Fixed keyboard accessibility for file picker button: keyboard users can now tab to focus on `fileInput()` widget ([#2929](https://github.com/rstudio/shiny/issues/2929)).
```

Comment:
A few things about NEWS items:

* Should say "`Closed #2929: ....... (#2937)`.
* Instead of including the URL to the issue or PR, this should just be `#2929` and `#2937`. This is because including the entire URL has caused problems on CRAN's build/check systems.

Now that I look at it, the previous two NEWS items should also have references to the issues and PRs.